assets/src/apps/authoring/components/RightMenu/PartPropertyEditor.tsx

```diff
@@ -345,6 +345,17 @@ export const PartPropertyEditor: React.FC<Props> = ({
     <div
       className={`component-tab p-3 overflow-hidden part-property-editor ${selectPartType}-part-property`}
     >
+      {selectPartType === 'janus-fill-blanks' && (
+        <Alert variant="info" className="part-documentation">
+          <a
+            href="https://etx-tech.notion.site/FITB-Component-New-20862d4b114580948615d0ccd2706450"
```

Comment:
These are the hardcoded link that contains the how to use the component. As my conversation with the team, this won't change. In future, they might be a different ticket to include this for other component as well once they have links for all the other component. During that time, we can make this generic and may be add a new attribute in the custom property where they can add the link.